src/app/pages/Info/Info.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import CardInfo from '../../components/CardInfo/CardInfo';
+import Header from '../../components/Header/Header';
+import { SPROUTS } from '../../lib/sprouts';
+import { useParams } from 'react-router';
+
+export default function Info(): JSX.Element {
+  const { id }: { id: string } = useParams();
+
+  const [sprouts] = useState(SPROUTS);
+
+  function handleOnClick() {
+    console.log('Add');
+  }
+  return (
+    <main>
+      <Header children="Info" onClick={() => history.back()} />
+      {sprouts
+        .filter((sprout) => sprout.id === id)
```

Comment:
I recommend moving this to a variable between line 10 and 12, which will get re-calculated on every re-render